doc/stdlib/arrays.rst

```diff
@@ -616,6 +616,33 @@ Indexing, Assignment, and Concatenation
 
    See also ``circshift``\ .
 
+.. function:: circcopy!(dest, src)
+
+   .. Docstring generated from Julia source
+
+   Copy ``src`` to ``dest``\ , indexing each dimension modulo its length. ``src`` and ``dest`` must have the same size, but can be offset in their indices; any offset results in a (circular) wraparound. If the arrays have overlapping indices, then on the domain of the overlap ``dest`` agrees with ``src``\ .
+
+   .. doctest::
+
+       julia> src = reshape(collect(1:16), (4,4))
+       4×4 Array{Int64,2}:
+        1  5   9  13
+        2  6  10  14
+        3  7  11  15
+        4  8  12  16
+
+       julia> dest = OffsetArray(Int, (-1,1));
```

Comment:
I just got rid of it, as (1) I was not successful in figuring out hiding, and (2) the printing of the typename just didn't seem to benefit from having TestHelpers in it. I did move the OAs module to TestHelpers.jl, however.